.travis.yml

```
@@ -17,18 +17,24 @@ before_install:
   - export PATH=$PATH:$HOME/.local/bin
 jobs:
   include:
-  - stage: Sanity check and tests
-  # Check generated contents are up to date and code is formatted.
+  - name: "Sanity checks and tests"
+    stage: test
+    # Check generated contents are up to date and code & docs files are formatted.
     script: make --always-make format generate-in-docker && git diff --exit-code
-  # Build Prometheus Operator rule config map to rule file crds cli tool
+    # Build Prometheus Operator rule config map to rule file CRDs cli tool.
```

this is a nit but since we are already fixing the wording :D 

```suggestion
    # Build Prometheus Operator rule config map to rule file CRDs CLI tool.
```